src/main.js

```diff
@@ -400,6 +400,13 @@ let load = loader({
               stability: cfg.app.stability,
               expires: taskcluster.fromNow('36 hours'),
               description: cfg.app.description,
+              actions: [{
+                name: 'kill',
+                title: 'Kill',
+                context: 'worker',
+                url: 'https://ec2-manager.taskcluster.net/v1/region/:workerGroup/instance/:workerId',
```

Comment:
This should probably be a configuration option -- for example, if the staging provisioner talks to a staging ec2-manager, we would want a different URL here for the two environments.

Reply:
In fact, it's already available at `cfg.ec2Manager`.

Reply:
Good idea, thanks for pointing that out.